crypto/evp/p_lib.c

```diff
@@ -684,8 +698,16 @@ int EVP_PKEY_print_private(BIO *out, const EVP_PKEY *pkey,
 int EVP_PKEY_print_params(BIO *out, const EVP_PKEY *pkey,
                           int indent, ASN1_PCTX *pctx)
 {
+    const char *pq = OSSL_SERIALIZER_Parameters_TO_TEXT_PQ;
+    int wd = OSSL_SERIALIZER_Parameters_WD;
+    int ret = OSSL_SERIALIZER_EVP_PKEY_to_bio(out, pkey, pq, wd);
+
+    if (ret != -2)
+        return ret;
+
     if (pkey->ameth && pkey->ameth->param_print)
         return pkey->ameth->param_print(out, pkey, indent, pctx);
```

Comment:
Is this line correct?

Reply:
Er... Why wouldn't it be?

Reply:
I was thinking that the new code above it effectively does the same thing - guess I am wrong :)

Reply:
These lines are conserved for legacy code.  The new code *only* deals with keys in providers.

Reply:
...  so I guess I should add a TODO comment...